src/content/reference/react/index.md

```diff
@@ -72,23 +73,23 @@ function ChatRoom({ roomId }) {
   // ...
 ```
 
-Effects are an "escape hatch" from the React paradigm. Don't use Effects to orchestrate the data flow of your application. If you're not interacting with an external system, [you might not need an Effect.](/learn/you-might-not-need-an-effect)
+Les Effets sont une "issue de secours" du paradigme de React. N'utilisez pas les Effets pour orchestrer le flux de données de votre application. Si vous n'intéragissez pas avec un système externe, [Vous n'avez pas besoin d'un Effet.](/learn/you-might-not-need-an-effect)
 
-There are two rarely used variations of `useEffect` with differences in timing:
+Il existe deux variantes rarement utilisées de `useEffect` avec des différences de timing :
 
-* [`useLayoutEffect`](/reference/react/useLayoutEffect) fires before the browser repaints the screen. You can measure layout here.
-* [`useInsertionEffect`](/reference/react/useInsertionEffect) fires before React makes changes to the DOM. Libraries can insert dynamic CSS here.
+* [`useLayoutEffect`](/reference/react/useLayoutEffect) se déclenche avant que le navigateur ne repeigne l'écran. Vous pouvez mesurer la mise en page ici.
+* [`useInsertionEffect`](/reference/react/useInsertionEffect) se déclenche avant que React ne fasse des changements dans le DOM. Les bibliothèques peuvent insérer du CSS dynamique ici.
 
 ---
 
-## Performance Hooks {/*performance-hooks*/}
+## Les Hooks de performance {/*performance-hooks*/}
 
-A common way to optimize re-rendering performance is to skip unnecessary work. For example, you can tell React to reuse a cached calculation or to skip a re-render if the data has not changed since the previous render.
+La manière courante d'optimiser la performance de réaffichage est d'éviter les tâches inutiles. Par exemple, vous pouvez demander à React de réutiliser la valeur mise en cache ou d'éviter un réaffichage si la donnée n'a pas changé depuis l'affichage précedent.
```

Comment:
```suggestion
Une manière courante d’optimiser la performance de réaffichage consiste à s’épargner des tâches superflues. Par exemple, vous pouvez demander à React de réutiliser un calcul mis en cache ou d'éviter un nouveau rendu si la donnée n'a pas changé depuis le rendu précédent.
```

- Attention au mot-à-mot, le "est" entre deux propositions c'est inélégant. Ne pas hésiter à reformuler un peu.
- Le premier "réaffichage" est bien ("rerendu", blerk) car on est sur un verbe, ça reste concis. Le deuxième est un nom, "nouveau rendu" dans le contexte de la phrase sera mieux, et on utilise souvent "rendu" pour "render" en tant que nom dans les trads ici.

Reply:
> Le premier "réaffichage" est bien ("rerendu", blerk) car on est sur un verbe, ça reste concis. Le deuxième est un nom, "nouveau rendu" dans le contexte de la phrase sera mieux, et on utilise souvent "rendu" pour "render" en tant que nom dans les trads ici.

exactement j'avais d'énorme doute sur la VF de render/re-render, je ne savais pas quoi mettre. je pense que c'est encore mieux d'ajouter certains mots dans le glossaire. qu'en pensez vous 🙂 ?

Reply:
J'en pense que le Glossaire a déjà une entrée de base + une section dédiée sur ces sujets 😉